.github/dependabot.yml

```diff
@@ -0,0 +1,27 @@
+version: 2
+updates:
+  # glean-core
+  - package-ecosystem: "cargo"
+    directory: "/glean-core"
+    schedule:
+      interval: "daily"
+    ignore:
+      - dependency-name: "rkv"
+
+  # glean-ffi
+  - package-ecosystem: "cargo"
+    directory: "/glean-core/ffi"
+    schedule:
+      interval: "daily"
+
+  # glean benchmark
+  - package-ecosystem: "cargo"
+    directory: "/glean-core/benchmark"
+    schedule:
+      interval: "daily"
+
+  # glean-python
+  - package-ecosystem: "pip"
+    directory: "/glean-core/python"
+    schedule:
```

Comment:
Let add: `ignore: glean_parser`, since I think we always want to update glean_parser in lock step in a few different places, and we don't want dependabot telling us when we want to do that one.

Reply:
Ah, I forgot that it is a dependency, will add it